octoprint/umbrel-app.yml

```diff
@@ -0,0 +1,28 @@
+manifestVersion: 1
+id: octoprint
+category: Networking
+name: OctoPrint
+version: "1.8.7"
+tagline: The snappy web interface for your 3D printer.
+description: >-
+  OctoPrint provides a snappy web interface for controlling consumer 3D printers.
+releaseNotes: >-
+  Improvements
+   Add support for Python 3.11 by bumping up the version of a dependency.
+  Bug fixes
+    Do not send None to Locale.parse if no match can be made between the request's accepted languages and the installed languages. Fixes a 500 server error on the API in case of Babel being at of version of 2.12 or higher.
+developer: OctoPrint
+website: https://octoprint.org/
+repo: https://github.com/OctoPrint/OctoPrint
+support: https://github.com/OctoPrint/OctoPrint/issues
+port: 5000
```

Comment:
I just checked and this port is already taken by [SatSale](https://github.com/getumbrel/umbrel-apps/blob/master/satsale/umbrel-app.yml). You can choose any free port for this (5003 is available for example). Do a global search of the umbrel-apps repo for `port: <your-port-choice>` and see if it is already taken in an apps `umbrel-app.yml`.

You don't have to change anything else for this to work. It is not the same thing as `APP_PORT` for the `app_proxy` service.

Reply:
done !